includes/class-wc-taxjar-customer-sync.php

```diff
@@ -146,44 +146,111 @@ public function add_customer_meta_fields( $user ) {
 		endforeach;
 	}
 
+	/**
+     * Saves tax exemption user meta if necessary
+     *
+	 * @param $user_id - Id of user to update
+	 */
 	public function save_customer_meta_fields( $user_id ) {
 		if ( ! apply_filters( 'taxjar_current_user_can_edit_customer_meta_fields', current_user_can( 'manage_woocommerce' ), $user_id ) ) {
 			return;
 		}
 
```

Comment:
Refactored this section to make the logic easier to follow. Also, it is no longer validating that the exemption settings on the user have changed at this point, instead it relies on the logic in the should_sync method of the record.